packages/versioning/test/versioned-dependencies.test.ts

```diff
@@ -912,17 +892,14 @@ describe("versioning: dependencies", () => {
         @useDependency(Lib.One.Versions.v1)
         @test("MyService")
         namespace MyOrg.MyService {
-
         }
-
         @versioned(Versions)
         namespace Lib.One {
           enum Versions { v1: "v1" }
         }
         
         @useDependency(Lib.One.Versions.v1)
         namespace Lib.Two { }
-
```

Comment:
I think this was all autoformatter... not sure why it did this.

Reply:
Weird since this is inside strings. I noticed but didn't mind either way.